src/components/Input.scss

```diff
@@ -25,23 +19,14 @@ $form-input-height: $su-xlarge !default;
 $form-input-bg-color: $cu-foreground--dark !default;
 
 // * 2. Base Input
+//
+// a. Override styles on base input[type] tags
 
 .Input {
-  display: block;
-  position: relative;
-  padding-bottom: $su-medium;
-}
-
-.Input-label {
-  display: block;
-  padding-bottom: $su-xxsmall;
-  font-size: $tu-small-fontSize;
-  font-weight: $tu-semibold-fontWeight;
-}
-
-.Input-element {
   cursor: text;
-  height: $form-input-height;
+  display: block;
+  width: 100%;
+  height: $form-input-height !important; // a.
```

Comment:
I'm wondering if we should just update that static stylesheet or link directly to the one in classic instead of adding things like !important to the new CSS.  There were some changes in classic that should allow you to override with just a class

Reply:
here's what they're doing in LFE https://github.com/teamsnap/league-frontend/blob/5185b0925544976a12bb1f6849c48a1e58620d04/src/index.html#L7

Reply:
Oh, hmmm. That's a good point that we could just update the style in classic that is setting the height too small, since we want it to be that way in Classic too.